pkgs/by-name/me/mealie/package.nix

```diff
@@ -96,42 +109,32 @@ pythonpkgs.buildPythonApplication rec {
 
     substituteInPlace mealie/__init__.py \
       --replace-fail '__version__ = ' '__version__ = "v${version}" #'
-
-    substituteInPlace mealie/services/backups_v2/alchemy_exporter.py \
-      --replace-fail 'PROJECT_DIR = ' "PROJECT_DIR = Path('$out') #"
-
-    substituteInPlace mealie/db/init_db.py \
-      --replace-fail 'PROJECT_DIR = ' "PROJECT_DIR = Path('$out') #"
-
-    substituteInPlace mealie/services/backups_v2/alchemy_exporter.py \
-      --replace-fail '"script_location", path.join(PROJECT_DIR, "alembic")' '"script_location", "${src}/alembic"'
```

Comment:
Why is this removed ?
Did something changed in upstream that makes it irrelevant ?

Reply:
https://github.com/mealie-recipes/mealie/commit/a6cbf1308e36a91ce67a2af354b6f005ddb095fc made this unnesseary.